models/apartment.rb

```diff
@@ -0,0 +1,32 @@
+class Apartment < ActiveRecord::Base
+  # AR classes are singular and capitalized by convention
+  has_many :tenants
+end
+
+#   attr_accessor :address
```

Comment:
You can save yourself some typing by writing `attr_accessor` once, followed by a comma separated list of attributes.